src/app/Common/DAL/BaseFilter.cs

```diff
@@ -3,5 +3,6 @@
     public class BaseFilter
     {
         public string Id { get; set; }
+        public bool EmptyFilterAllowed { get; set; }
```

Comment:
I prefer AllowEmptyFilter or IsEmptyFilterAllowed. But it is not critical

Reply:
> Looks good. Did you test existing functionality?

I've ran into that problem on Servgrow. He're same logic, so yeah.
Also implemented "FindAsync" method for testing purposes in this solution (removed before comitting): error reproduced. That's why I've made this little fix. Also implemented AllowEmptyFilter for the reasons I've mentioned in the PR message.

As for the name: I've named it using my personal taste, It's not critical to rename it. You can check MSDN naming convention: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members. TLDR: `"Optionally, you can also prefix Boolean properties with "Is", "Can", or "Has", but only where it adds value"`. I think `V + ed => participle` describes this property being a boolean. As far as I know you, we will not agree with points of each other. So I think we need Sergey's opinion aswell